episodes/09-open-and-plot-vector-layers.Rmd

```diff
@@ -0,0 +1,195 @@
+
+:::
+
+First we need to load the packages we will use in this lesson. We will use the package `tidyverse` with which you are already familiar from the previous lesson. In addition, we need to load the [`sf`](https://r-spatial.github.io/sf/) package for working with spatial vector data. 
+
+```{r lib, message=FALSE}
+library(tidyverse)  # wrangle, reshape and visualize data
+library(sf)         # work with spatial vector data
+```
+
+::: callout
+
+# The `sf` package
+
+`sf` stands for Simple Features which is a standard defined by the Open Geospatial Consortium for storing and accessing geospatial vector data. PostGIS, for instance, uses the same standard, so PostGIS users might recognise the functions used by the `sf` package. 
+
+:::
+
+## Import shapefiles
+
+Let's start by opening a shapefile. Shapefiles are a common file format to store spatial vector data used in GIS software. We will read a shapefile with the administrative boundary of Delft with the function `st_read()` from the `sf` package. 
+
+```{r}
+boundary_Delft <- st_read("data/delft-boundary.shp", quiet = TRUE)
+```
+
+::: callout
+
+# All `sf` functions start with `st_`
+
+Note that all functions from the `sf` package start with the standard prefix `st_` which stands for Spatial Type. This is helpful in at least two ways:  
+
+1. it makes the interaction with or translation to/from software using the simple features standard like PostGIS easy, and  
+2. it allows for easy autocompletion of function names in RStudio.
+
+:::
+
+## Spatial Metadata
+
+By default (with `quiet = FALSE`), the `st_read()` function provides a message with a summary of metadata about the file that was read in. To examine the metadata in more detail, we can use other, more specialised, functions from the `sf` package.  
+
+The `st_geometry_type()` function, for instance, gives us information about the geometry type, which in this case is `POLYGON`.
+
+```{r}
+st_geometry_type(boundary_Delft)
+```
+
+The `st_crs()` function returns the coordinate reference system (CRS) used by the shapefile, which in this case is ``r st_crs(boundary_Delft)$Name`` and has the unique reference code ``r paste0("EPSG: ", st_crs(boundary_Delft)$epsg)``. 
+
+```{r}
+st_crs(boundary_Delft)
+```
+
+::: callout
+
+# Examining the output of `st_crs()`
+
+As the output of `st_crs()` can be long, you can use `$Name` and `$epsg` after the `crs()` call to extract the projection name and EPSG code respectively.
+
+:::
+
+The `st_bbox()` function shows the extent of the layer. As `WGS 84` is a **geographic CRS**, the extent of the shapefile is displayed in degrees.
+
+```{r}
+st_bbox(boundary_Delft)
+```
+
+We need a **projected CRS**, which in the case of the Netherlands is typically the Amersfort / RD New projection. To reproject our shapefile, we will use the `st_transform()` function. For the `crs` argument we can use the EPSG code of the CRS we want to use, which is `28992` for the `Amersfort / RD New` projection. 
```

Comment:
Would it be convenient to add here a resource indicating where to check the EPSG code of any CRS?
After I took the workshop and I reviewed the script, I missed where to check the EPSG code of a CRS different to the Dutch one.

(I found this website particularly convenient: https://epsg.io)

Reply:
@ana-5r, I assume this is mentioned in the 5-pager you are preparing